torchtext/__init__.py

```diff
@@ -1,21 +1,15 @@
 import os
 
+from torchtext import _extension  # noqa: F401
```

Comment:
Can you add a note that this is done for side-effect and it has to happen before anything?

Reply:
Done